CHANGELOG.md

```diff
@@ -7,6 +7,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [16.0.0] - 2021-11-30
+### Added
+- Support deploying Ceph Pacific
+- ceph-salt-formula: Add hosts with admin label (#457)
+- ceph-salt-formula: give explicit IP addr when adding host (#465)
+
+### Fixed
+- Tox: Fix issues when running tox on Python 3.8 (#457)
+- Remove Travis CI (#459)
+- .github/workflows: add tox and more python versions (#458)
+
+## [15.2.16] - 2021-11-26
+### Added
+- _modules/ceph_salt: log SSH commands (#464)
+
+### Fixed
+- Move ceph-salt-registry-json creation to container.sls (#467)
+- Use `cephadm registry-login --registry-json` (#467)
+- Rely on cephadm package for cephadm user creation (#461)
+- README.md: Fix broken cephadm link (#455)
+- tests: add source dir as real directory to fake fs (#462)
+- .github/workflows/linting: use ceph_salt instead old name (#458)
```

Comment:
These 15.2.16 entries are PRs that got merged to master and then cherry-picked to octopus?

Reply:
Correct